performance/TestResults.js

```diff
@@ -1,4 +1,4 @@
-/* eslint no-console: ["error", { allow: ["warn", "error"] }] */
+/* eslint-disable no-console, no-undef */
```

Comment:
Maybe you could leave one line empty below to keep the same style.

Reply:
Sure